index.html

```diff
@@ -940,19 +945,37 @@
 
 ### Rights {#data-rights}
 
-To the extent possible, [=people=] should retain a certain number of rights over their data, and
-these rights should be facilitated by the [=parties=]. While data rights are a self-governing
-mechanism that is not sufficient to support [=privacy=], they do complement self-determination and
-help improve accountability. Such rights should include:
+[=People=] retain certain rights over data about themselves, and these rights should be facilitated by their [=user agent=] and the [=parties=]. While data rights alone are not sufficient to support [=privacy=], they do support self-determination and help improve accountability. Such rights include:
 
 * The right to <em>access</em> [=data=] about oneself.
-* The right to <em>erase</em> [=data=] about oneself, with or without terminating the service (though
-  what [=data=] can be erased may differ between those two cases).
-* The right to <em>port</em> information that one has stored with a given [=party=] in such a way
-  that it can easily be reused or transferred elsewhere.
+
+  This right includes being able to discover data about oneself (no secret databases) and to review what information has been collected or inferred.
+
+* The right to <em>erase</em> [=data=] about oneself, sometimes described as a <em>right to be forgotten</em>. 
+
+  The right to erase applies whether or not terminating use of a service altogether, though what [=data=] can be erased may differ between those two cases. On the Web, [=people=] may wish to erase data on their device, on a server, or both, and the distinctions may not always be clear.
+
+* The right to <em>port</em> [=data=], including data one has stored with a [=party=], so it can easily be reused or transferred elsewhere.
+
+  Portability is needed to realize the ability for [=people=] to make choices about services with different data practices. Standards for interoperability are essential for effective re-use.
+
 * The right to <em>correct</em> [=data=] about oneself, to ensure that one's [=identity=] is
   properly reflected in a system.
 
+* The right to <em>be free from automated decision-making</em> based on data about oneself.
+
+  For some kinds of decision-making with substantial consequences, there is a privacy interest in being able to exclude oneself from automated profiling.
+
+  <aside class="issue">
+    Should we settle on examples of the kind of automated decision-making that is impactful enough that one can opt out of it? Are there direct ways to apply this to the Web generally?
+  </aside>  
```

Comment:
I see this was also discussed [a couple weeks ago](https://github.com/w3ctag/privacy-principles/blob/main/meetings/2022-02-23-minutes.md#pr-136) … I'm not sure the "automated" is important here, even though it's the target of some current laws. We want people to be able to practically interrogate and challenge decisions made about them. Automation can make that hard by breaking the assumption that an organization will need a similar number of customer service staff as decision-makers, or by adding more steps between customer service and the programmers with the power to change decisions. But an organization can decide to cut its customer service even without automating away its decision-makers, and the effect is just as bad on the victims of the decisions. So ... let's focus on holding the decision-making accountable?

Reply:
I think @samuelweiler also raised this concern in the meeting. I think the privacy right is about freedom from certain decisions made about them based on data about them -- it's a privacy right because you might object to the data that was collected about you or to the inferences drawn from some piece of data. That is often described or operationalized as freedom from automated or algorithmic decision-making, but it maybe doesn't actually matter much whether a decision was made by a person or not.

Hopefully examples will be illustrative here.

Reply:
I agree that automation is a contributing factor but not necessarily essential. However, I worry that removing the automation may put us in a position to be asking too much (or at least, more than we reasonably might). Without automation, this is essentially a right to freedom from data-based decisions, which is essentially a right to freedom from [legibility](https://bookshop.org/books/seeing-like-a-state-how-certain-schemes-to-improve-the-human-condition-have-failed/9780300246759) and [categorisation](https://bookshop.org/books/sorting-things-out-classification-and-its-consequences/9780262522953). We're pretty much saying that people should be free from bureaucratic management… It's not that I want to stop anyone from going there, but we might have trouble operationalising this.

My understanding of the motivation for focusing on the automation side is that automation is used to entrench and opacify existing uninterrogated structures, to decrease the cost of discrimination, and to increase the cost of fighting back. If you need thousands of clerical workers, there's a cost, they might blow the whistle, they might develop street-level bureaucratic flexibility, etc.

Maybe the trick is, instead of advocating against the automation of decision-making, we should advocate for decision-making (automated or not) to have certain positive properties? I'm not an expert in this area, but I think that that's what the code-as-law people are thinking about.